src/__tests__/test_distillation_tower.js

```diff
@@ -0,0 +1,43 @@
+import MLR from '..';
+
+var fs = require('fs');
+var path = require('path');
+
+var Papa = require('papaparse');
+
+// Dataset -> https://openmv.net/info/distillation-tower
+
+describe('multivariate linear regression', () => {
+  it('should work with large dataset with intercept', () => {
+    const ymatrix = fs.readFileSync(path.resolve(path.join('./data/ydistillationjs.csv')), 'utf8');
```

Comment:
you can remove calls to the `path` module. `fs` already converts relative paths using `path.resolve` and `path.join` should be used to concatenate paths.